src/growth-pandora.R

```diff
@@ -318,27 +352,44 @@ generateParameterFile <- function(Iteration, FireID, season, Lat, Lon, data) {
     str_c("Init_hour 13"),
     str_c("FFMC_Method 5"),
     str_c("Threads 1"),
-    if(useWindGrid){ WindGridParameterStrings } else NA,
-    str_c("Greenup ", GreenUp %>% filter(Season %in% c(season, NA)) %>% arrange(Season) %>% pull(GreenUp) %>% pluck(1) %>% as.numeric),
-    if(setGrassCuring){ str_c("%Curing_GridFile ", Curing %>% filter(Season %in% c(season, NA)) %>% arrange(Season) %>% pull(FileName) %>% pluck(1)) } else NA,
-    if(setFuelLoad){ str_c("Fuel_Load_GridFile ", FuelLoad %>% filter(Season %in% c(season, NA)) %>% arrange(Season) %>% pull(FileName) %>% pluck(1)) } else NA,
+    if (useWindGrid) {
+      WindGridParameterStrings
+    } else {
+      NA
+    },
+    str_c("Greenup ", GreenUp %>% filter(Season %in% c(season, NA)) %>% arrange(Season) %>% pull(GreenUp) %>% pluck(1) %>% as.numeric()),
```

Comment:
I wonder if the auto-formatter didn't catch this. Should pipes in arguments be spread across lines? A couple more instances below.

Reply:
Solved